hack/quickstart/init-master.sh

```diff
@@ -70,31 +70,36 @@ function init_master_node() {
     exit 1
 }
 
-# This script can execute on a remote host by copying itself + kubelet service unit to remote host.
+# This script can execute on a remote host by copying itself + kubelet service unit + auto_approver to remote host.
 # After assets are available on the remote host, the script will execute itself in "local" mode.
 if [ "${REMOTE_HOST}" != "local" ]; then
     # Set up the kubelet.service on remote host
     scp -i ${IDENT} -P ${REMOTE_PORT} ${SSH_OPTS} kubelet.master core@${REMOTE_HOST}:/home/core/kubelet.master
     ssh -i ${IDENT} -p ${REMOTE_PORT} ${SSH_OPTS} core@${REMOTE_HOST} "sudo mv /home/core/kubelet.master /etc/systemd/system/kubelet.service"
 
-    # Copy self to remote host so script can be executed in "local" mode
+    # Copy the auto approver to remote host and execute it for 1h
+    scp -i ${IDENT} -P ${REMOTE_PORT} ${SSH_OPTS} ../auto_approver.sh core@${REMOTE_HOST}:/home/core/auto_approver.sh
+    ssh -i ${IDENT} -p ${REMOTE_PORT} ${SSH_OPTS} core@${REMOTE_HOST} "/home/core/auto_approver.sh 3600" &
```

Comment:
Why not have bootkube auto-approve the first node which tries to join? In case people didn't want to use the tls bootstrap -- you could have it auto-sign or wait for a node to join. Whichever comes first.

I'm a little wary of relying on more external components for the bootstrap process to work (especially a random backgrounded shell process).

If the intention is that we allow nodes to join for a longer period - I think it might be better to let bootkube start it's process after first node has joined (and it auto-approves), but once an api-server is available we deploy a pod/job/deployment for an autoapprover (not really sure what our desired functionality is here).